sdk/keyvault/keyvault-keys/test/public/list.spec.ts

```diff
@@ -93,7 +93,9 @@ describe("Keys client - list keys in various ways", () => {
     const keyName = testClient.formatName(`${keyPrefix}-${this!.test!.title}-${keySuffix}`);
     await client.createKey(keyName, "RSA");
     let totalVersions = 0;
-    for await (const page of client.listPropertiesOfKeyVersions(keyName).byPage()) {
+    for await (const page of client
+      .listPropertiesOfKeyVersions(keyName)
+      .byPage({ maxPageSize: 1 })) {
```

Comment:
If there's only one version of the test key, are we sure that reducing the page size to 1 will guarantee we iterate into a second page?

Reply:
Thanks, that's a good catch.

Reply:
It does look like the other two tests that were modified would iterate pages though, so it's up to you about whether to make a change here

Reply:
yeah, those ones are for different methods though, not `listPropertiesOfKeyVersions` -- the goal is to get coverage for all the paginated methods so I will fix this one.